scapy/sendrecv.py

```diff
@@ -323,9 +330,7 @@ def _process_packet(self, r):
                             self.noans += 1
                         sentpkt._answered = 1
                     break
-        if self._send_done and self.noans >= self.notans and not self.multi:
```

Comment:
To explain this code can be reached before the sending thread exists (because of the new time.sleep) in which case the sniffer is never closed